libnm-core/nm-setting-wpan.c

```diff
@@ -270,5 +357,33 @@ nm_setting_wpan_class_init (NMSettingWpanClass *klass)
 	                            G_PARAM_READWRITE |
 	                            G_PARAM_STATIC_STRINGS));
 
+	/**
+	 * NMSettingWpan:page:
+	 *
+	 * IEEE 802.15.4 channel page.
```

Comment:
`Since:`

Also, can you describe a bit more, about the valid range, and the default value -1.